src/components/Person/Person.jsx

```diff
@@ -1 +1,29 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+      {age && (
+        <p className="Person__age">
+          {`I am ${age}`}
+        </p>
+      )}
+
+      <p className="Person__partner">
+        {(() => {
+          const partnerRelationship = sex === 'f' ? 'husband' : 'wife';
+
+          return isMarried ? (
+            `${partnerName} is my ${partnerRelationship}`
+          ) : (
+            'I am not married'
+          );
+        }
+        )()}
```

Comment:
it's bad practice to create anonymous function in the template, better to move it to the variable